util/envDynawo.sh

```diff
@@ -1574,26 +1647,84 @@ create_distrib_with_omc() {
   # Deploy Dynawo
   deploy_dynawo
 
+  if [ "$DYNAWO_COMPILER" = "GCC" ]; then
+    gcc_version=$($DYNAWO_C_COMPILER -dumpversion)
+    if [ $(echo $gcc_version | cut -d '.' -f 1) -lt 5 ]; then
+      sed -i 's/FLAGS="$FLAGS -std=c++98"/FLAGS="$FLAGS -std=c++98 -D_GLIBCXX_USE_CXX11_ABI=0"/' $DYNAWO_DEPLOY_DIR/sbin/compileCppModelicaModelInDynamicLib
+    elif [ $(echo $gcc_version | cut -d '.' -f 1) -eq 5 -a $(echo $gcc_version | cut -d '.' -f 2) -lt 1 ]; then
+      sed -i 's/FLAGS="$FLAGS -std=c++98"/FLAGS="$FLAGS -std=c++98 -D_GLIBCXX_USE_CXX11_ABI=0"/' $DYNAWO_DEPLOY_DIR/sbin/compileCppModelicaModelInDynamicLib
+    fi
+  fi
+
+  copy_sources
+
   if [ ! -x "$(command -v zip)" ]; then
     error_exit "You need to install zip command line utility."
   fi
 
+  if [ "`uname`" = "Linux" ]; then
+    if [ -x "$(command -v chrpath)" ]; then
+      if [ "$DYNAWO_LIBRARY_TYPE" = "SHARED" ]; then
+        chrpath -d $DYNAWO_DEPLOY_DIR/lib/libamd.so
+        chrpath -d $DYNAWO_DEPLOY_DIR/lib/libbtf.so
+        chrpath -d $DYNAWO_DEPLOY_DIR/lib/libcolamd.so
+        chrpath -d $DYNAWO_DEPLOY_DIR/lib/libklu.so
+        chrpath -d $DYNAWO_DEPLOY_DIR/lib/libsuitesparseconfig.so
+      fi
+    fi
+  fi
+
+  if [ "`uname`" = "Darwin" ]; then
+    version=$($DYNAWO_DEPLOY_DIR/bin/dynawo --version | cut -d ' ' -f 1)
```

Comment:
I propose to put this large amount of apple specific script into a method

Reply:
Ok